.claude/commands/explain.md

```
@@ -0,0 +1,39 @@
+**Purpose**: Technical documentation and knowledge transfer
+
+---
+
+@include shared/universal-constants.yml#Universal_Legend
+
+## Command Execution
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)